File: oa-portal/problems/m4-warehouse-bay-audit/starters/main.cpp
#include <bits/stdc++.h>
using namespace std;

int longestAisle(const vector<int>& ids) {
    // write your code here

    return 0;
}

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n;
    if (!(cin >> n)) return 0;
    vector<int> ids(n);
    for (auto &x : ids) cin >> x;

    cout << longestAisle(ids) << '\n';
    return 0;
}

File: oa-portal/problems/m4-warehouse-bay-audit/starters/main.java
import java.io.*;
import java.util.*;

public class Main {

    static int longestAisle(int[] ids) {
        // write your code here

        return 0;
    }

    public static void main(String[] args) throws IOException {
        FastReader in = new FastReader();
        int n = in.nextInt();
        int[] ids = new int[n];
        for (int i = 0; i < n; i++) ids[i] = in.nextInt();

        System.out.println(longestAisle(ids));
    }

    static class FastReader {
        private final DataInputStream in = new DataInputStream(new BufferedInputStream(System.in, 1 << 16));
        int nextInt() throws IOException { return (int) nextLong(); }
        long nextLong() throws IOException {
            int b = in.read();
            while (b != '-' && (b < '0' || b > '9')) b = in.read();
            boolean neg = b == '-';
            if (neg) b = in.read();
            long v = 0;
            while (b >= '0' && b <= '9') { v = v * 10 + (b - '0'); b = in.read(); }
            return neg ? -v : v;
        }
    }
}
